src/main/java/com/buyanywhere/productcatalog/controllers/v1/CategoryController.java

```diff
@@ -1 +1,40 @@
+package com.buyanywhere.productcatalog.controllers.v1;
+import com.buyanywhere.productcatalog.exceptions.CategoryNotFoundException;
+import com.buyanywhere.productcatalog.models.Category;
+import com.buyanywhere.productcatalog.repositories.CategoryRepository;
+import org.springframework.web.bind.annotation.*;
 
+@RestController
+@RequestMapping("/v1/category")
+public class CategoryController{
+    private CategoryRepository repository;
+
```

Comment:
Please delete this blank lines

Reply:
The requested modifications have been sent